scripts/simpledvt/02-clone-nor.js

```diff
@@ -0,0 +1,474 @@
+  const srcVersion = semanticVersion.map((n) => n.toNumber())
+
+  log(`Source App:`, yl(srcAppName))
+  log(`Source App ENS:`, yl(srcAppFullName))
+  log(`Source App ID:`, yl(srcAppId))
+  log(`Source Contract implementation:`, yl(contractAddress))
+  log(`Source App version:`, yl(srcVersion.join('.')))
+  log.splitter()
+
+  const trgAppFullName = `${trgAppName}.${state.lidoApmEnsName}`
+  const trgAppId = namehash(trgAppFullName)
+  const trgProxyAddress = readStateAppAddress(state, `app:${trgAppName}`)
+  const trgAppArtifact = APP_ARTIFACTS[srcAppName] // get source app artifact
+  const trgApp = await artifacts.require(trgAppArtifact).at(trgProxyAddress)
+
+  // set new version to 1.0.0
+  const trgVersion = [1, 0, 0]
+  const contentURI = '0x' + Buffer.from(`ipfs:${ipfsCid}`, 'utf8').toString('hex')
+
+  log(`Target App:`, yl(trgAppName))
+  log(`Target App ENS:`, yl(trgAppFullName))
+  log(`Target App ID:`, yl(trgAppId))
+  log(`Target App proxy`, yl(trgProxyAddress))
+  log(`Target Contract implementation:`, yl(contractAddress))
+  log(`Target Content IPFS CID:`, yl(ipfsCid))
+  log(`Target Content URI:`, yl(contentURI))
+  log(`Target App version:`, yl(trgVersion.join('.')))
+
+  log.splitter()
+  const {
+    moduleName,
+    moduleType = 'curated',
+    targetShare = 1000,
+    moduleFee = 500,
+    treasuryFee = 500,
+    penaltyDelay,
+    easyTrackAddress,
+    easyTrackEVMScriptExecutor,
+    easyTrackFactories = {},
+  } = state[`app:${trgAppName}`].stakingRouterModuleParams
+  log(`Target SR Module name`, yl(moduleName))
+  log(`Target SR Module type`, yl(moduleType))
+  log(`Target SR Module fee`, yl(moduleFee))
+  log(`Target SR Module targetShare`, yl(targetShare))
+  log(`Target SR Module treasuryFee`, yl(treasuryFee))
+  log(`Target SR Module penaltyDelay`, yl(penaltyDelay))
+
+  if (!trgProxyAddress || (await web3.eth.getCode(trgProxyAddress)) === '0x') {
+    log.error(`Target app proxy is not yet deployed!`)
+    return
+  }
+
+  const trgRepoAddress = await resolveEnsAddress(artifacts, ens, trgAppId)
+
+  if (trgRepoAddress && (await web3.eth.getCode(trgProxyAddress)) !== '0x') {
+    log(`Target App APM repo:`, yl(trgRepoAddress))
+    log.error(`Target app is already deployed!`)
+    return
+  }
+
+  // preload voting and stakingRouter addresses
+  const votingAddress = readStateAppAddress(state, `app:${APP_NAMES.ARAGON_VOTING}`)
+  const tokenManagerAddress = readStateAppAddress(state, `app:${APP_NAMES.ARAGON_TOKEN_MANAGER}`)
+  const srAddress = readStateAppAddress(state, 'stakingRouter')
+
+  const kernel = await artifacts.require('Kernel').at(kernelAddress)
+  const aclAddress = await kernel.acl()
+  const acl = await artifacts.require('ACL').at(aclAddress)
+  const stakingRouter = await artifacts.require('StakingRouter').at(srAddress)
+  const apmRegistry = await artifacts.require('APMRegistry').at(state.lidoApmAddress)
+
+  const voteDesc = `Clone app '${srcAppName}' to '${trgAppName}'`
```

Comment:
I think we should make a more detailed description of the vote

Reply:
agree, it's ok for testnet, should be updated